kubernetes/docker-registry/templates/registry_daemonset.yaml

```
@@ -0,0 +1,39 @@
+    matchLabels:
+      {{- include "common.labels.matchLabels" . | nindent 6 }}
+      app.kubernetes.io/component: docker-registry
+  template:
+    metadata:
+      labels:
+        {{- include "common.labels.standard" . | nindent 8 }}
+        app.kubernetes.io/component: docker-registry
+    spec:
+      containers:
+      - name: registry
+        image: {{ include "common.images.image" ( dict "imageRoot" .Values.image "global" .Values.global ) }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)